syntax = "proto3";

import "google/protobuf/empty.proto";

service ChatService {
  rpc SendMessage (MessageRequest) returns (MessageResponse) {}
  rpc ReceiveMessage (google.protobuf.Empty) returns (stream MessageResponse) {}
}

message MessageRequest {
  string sender = 1;
  string message = 2;
}

message MessageResponse {
  string sender = 1;
  string message = 2;
}
